docs/toc.md

```diff
@@ -114,6 +114,7 @@
 ## [Runtime IDentifier catalog](core/rid-catalog.md)
 ## [.NET Core SDK Overview](core/sdk.md)
 ## [.NET Core CLI Tools](core/tools/index.md)
+## [.NET Core Additional Tools](core/additional-tools/index.md)
```

Comment:
this should be moved to line 149 before the [Porting from .NET Framework](core/porting/index.md) entry

the other two new topics should be under this toc entry with the `###`